benchmarks/infra/stage-1/main.tf

```diff
@@ -31,7 +31,7 @@ module "gke-infra" {
 
   registry_create = true
 
-  enable_private_endpoint = var.enable_private_endpoint
+  private_cluster_config = var.private_cluster_config
```

Comment:
README needs to be changed to reflect the variable change. Please use tfdocs to update it.00

Ideally, it would be best if this doesn't break the existing sample-terraform-vars that people are using when they don't modify it. Is that the case still?

Reply:
Thanks, ran `terraform-docs markdown table --output-file ./README.md` .

Unfortunately this does break the previous tfvar files for anyone who was setting `enable_private_endpoint`. That's not the case in sample-terraform.tfavrs but of course could be for anyone else. Want me to leave the old variable around & use it in the new one?

Reply:
Yes, let's leave the old variable around in case anyone was relying on that and use it to set the value when specified.